cli/command/container/opts.go

```diff
@@ -776,15 +803,18 @@ func applyContainerOptions(n *opts.NetworkAttachmentOpts, copts *containerOption
 }
 
 func parseNetworkAttachmentOpt(ep opts.NetworkAttachmentOpts) (*networktypes.EndpointSettings, error) {
+	Netmode := container.NetworkMode(ep.Target)
```

Comment:
nit: variable should start with lowercase

Reply:
Yes，I have fixed it.
Please check it.